policies/modules/ROOT/pages/policies-included-apply.adoc

```diff
@@ -28,14 +28,18 @@ You can apply a policy either to the entire API (default) or to specific methods
 To apply a policy for Mule Gateway, a Mule application must first be deployed with an xref:connectors::http/http-listener-ref.adoc[HTTP-based or HTTPS-based flow] that is linked to the API through autodiscovery.
 ====
 
-To apply a policy for Connected Mode or Mule Gateway:
+To apply a policy for Flex Gateway running in Connected Mode or Mule Gateway:
```

Comment:
```suggestion
To apply a policy for Flex Gateway running in Connected Mode, or for Mule Gateway:
```

Otherwise it implies that Flex Gateway is running in Mule Gateway.